bin/inference/pycbc_inference_plot_samples

```diff
@@ -78,11 +78,20 @@ for i, arg in enumerate(parameters):
                             thin_interval=opts.thin_interval,
                             thin_end=opts.thin_end)
         y = transforms.apply_transforms(y, cs)
-        axs[i].plot(y[arg], alpha=0.25)
+        x = range(opts.thin_start, opts.thin_end + 1)
+
+        axs[i].plot(x, y[arg], alpha=0.25)
 
         # y-axis label
         axs[i].set_ylabel(labels[i])
 
+# x-axis label
+if opts.thin_interval is not None:
+    axs[-1].set_xlabel("Intervals [%s iteration(s) per interval]" % opts.thin_interval)
+# Default to old behavior
+else :
+    axs[-1].set_xlabel("Autocorrelation Lengths [%s iteration(s) per interval]" % fp.attrs['acl'])
```

Comment:
Looks like this isn't going to describe what is actually being plotted when `fp.attrs['acl']` != 1. I see that in line 83, samples from all iterations are being plotted and not the ones after every `fp.attrs['acl']` interval. So I think labeling this one as just `iterations` would be right.